pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -79,11 +79,12 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		false,
 		"retain nodes for debugging when cluster creation fails",
 	)
-	cmd.Flags().DurationVar(
+
+	cmd.Flags().IntVar(
 		&flags.Wait,
 		"wait",
-		time.Duration(0),
-		"wait for control plane node to be ready (default 0s)",
+		0,
```

Comment:
`10` is not a valid input to `time.ParseDuration()` which is used by pflag and cobra.
```
d, err := time.ParseDuration("10")
fmt.Println(err, d) // output: `time: missing unit in duration "10" 0s`
```

also, this is a breaking change on the kind CLI, as users who used to pass `10s` for `--wait` (a correct `time.Duration` input), will no longer be able to.

Reply:
Ok, I see, fully agree.  So  this is not a code bug but more a lack of clear documentation, how  **--wait** option format should look like.